x/epochstorage/keeper/params.go

```diff
@@ -142,10 +142,19 @@ func (k Keeper) GetNextEpoch(ctx sdk.Context, block uint64) (nextEpoch uint64, e
 func (k Keeper) GetCurrentNextEpoch(ctx sdk.Context) (nextEpoch uint64) {
 	epochBlocks := k.EpochBlocksRaw(ctx)
 	details, found := k.GetEpochDetails(ctx)
-	nextEpoch = details.StartBlock + epochBlocks
-	if !found {
-		utils.LavaFormatPanic("blabla", nil)
+	if details.EarliestStart == details.StartBlock {
```

Comment:
this happens only for a new chain.
the problem is that using the fixated params on genesis causes trouble so if this is the first epoch we can do this.
otherwise it will use the fixated params